### PR TITLE
instruction control enablement : start/stop all support

### PR DESCRIPTION
**Pull request by lkammath**

### CODE REVIEW

File: src/dll/edbgCipDll.C [hunk 1]
```diff
@@ -34,13 +35,20 @@
 #include <ecmdReturnCodes.H>
 #include <ecmdDataBuffer.H>
 #include <ecmdSharedUtils.H>
+#include <ecmdChipTargetCompare.H>
```
Comment (thejta) [inline src/dll/edbgCipDll.C]: Why did this need to be added, I don't see anything in the code that requires it?

File: src/dll/edbgCipDll.C [hunk 2]
```diff
@@ -22,6 +22,7 @@
 //--------------------------------------------------------------------
 // Includes
 //--------------------------------------------------------------------
+
```
Comment (thejta) [inline src/dll/edbgCipDll.C]: No need for the empty line

Comment (lkammath) [reply src/dll/edbgCipDll.C]: done

**Comment by lkammath** [comment]
Fixed both review comments

**Pull request merged**